packages/front-end/components/Experiment/ChanceToWinColumn.tsx

```diff
@@ -76,6 +77,22 @@ export default function ChanceToWinColumn({
         className
       )}
     >
+      {suspiciousChange && (
+        <div>
+          <div className="mb-1 d-flex">
+            <span className="badge badge-pill badge-warning">
+              Suspicious Result
+            </span>
+            <Tooltip
+              body={`A suspicious result occurs when the percent change is equal to or greater than your maximum percent change (${
+                metric.maxPercentChange * 100
+              }%).`}
+            >
+              <FaQuestionCircle />
```

Comment:
Instead of a separate tooltip icon, I might just wrap the Tooltip around the badge itself.